packages/cli/src/commands/apps/update.ts

```diff
@@ -17,7 +17,7 @@ import {
 	InputDefsByProperty,
 } from '@smartthings/cli-lib'
 import { addPermission } from '../../lib/aws-utils'
-import { chooseApp, smartAppHelpText, tableFieldDefinitions } from '../../lib/commands/apps-util'
+import { chooseApp, tableFieldDefinitions } from '../../lib/commands/apps-util'
```

Comment:
Do these need the `.js` extension?

Reply:
Amazing catch. They will but we can fix them when we update and move them since they aren't being compiled currently. I'm not sure how my regex search and replaces missed them!